src/python/tensorflow_cloud/core/containerize.py

```diff
@@ -117,15 +119,26 @@ def get_docker_image(
         """
         raise NotImplementedError
 
-    def get_generated_files(self):
-        return [self.docker_file_path, self.tar_file_path]
+    def get_generated_files(self, return_descriptors=False):
+        """Get generated file paths and/or descriptors for generated files.
+
+        Args:
+            return_descriptors: Whether to return descriptors as well.
+        """
```

Comment:
Can you add a returns section to this doc string as well?

Reply:
https://github.com/tensorflow/cloud/pull/218/commits/10335ecd4b564384210fe250a7019f946d723008